### ISSUE TITLE
Content clipping with rotation

### ISSUE THREAD

**Issue by iangilman** [openseadragon/openseadragon#463]
@dom96 reported:

> There is one issue that I found and I am not sure what the best way to fix it is. Namely, rotating an image which has a width which is considerably higher than its height by 90 degrees results in a section of the image not being visible. Both in the navigator and in the main viewer. My suspicion is that somewhere the drawer (or perhaps some other class) stores data about the image size and it simply needs to be rotated. I have tried finding the location of this but after some time decided that it may be best to ask here and create a PR for what I have got done so far. If someone could point me in the right direction I would really appreciate it.
> 
> I tried it with the demo and I can reproduce it. It's a bit difficult to notice with that image. Try panning to the top of the image after rotating the image 90 or 180 degrees, keep an eye on the display region (red rectangle) in the navigator, you will notice that the rectangle goes past the top of the thumbnail because the image in the thumbnail is actually bigger than the navigator height. You can also zoom out completely at which point you will note that the whole image is not shown.

**Comment by iangilman** [comment]
Seems related to #567.

**Comment by iangilman** [comment]
@avandecreme Should this be closed now, after #782?

**Comment by avandecreme** [comment]
Yes, it was a duplicate of #567 I think (or rather the other way around).

**Comment by iangilman** [comment]
Excellent; closing.
